openstudiocore/src/model/FoundationKiva.cpp

```diff
@@ -308,6 +332,96 @@ namespace detail {
   SurfaceVector FoundationKiva_Impl::surfaces() const {
     return getObject<ModelObject>().getModelObjectSources<Surface>(Surface::iddObjectType());
   }
+  
+  // TODO: this field shouldn't even exist in the IDD
```

Comment:
@joseph-robertson are you going to remove this field?

Reply:
This PR (and https://github.com/NREL/OpenStudio/pull/3565) aren't ready for review yet.